integration

```diff
@@ -508,6 +508,7 @@
   "mampfes/ha_freeair_connect",
   "mampfes/hacs_dwd_pollenflug",
   "mampfes/hacs_waste_collection_schedule",
+  "mandarons/ha-bouncie",
```

Comment:
<https://github.com/mandarons/ha-bouncie/blob/main/custom_components/bouncie/usage.py>
That seems invasive.
Please point me to the documentation for it, it was not in your README.

Reply:
It is described in this file: https://github.com/mandarons/ha-bouncie/blob/main/USAGE.md. It is based on analytics.home-assistant.io.

Reply:
No one is going to stumble upon that file.
The entire logic is not needed. <https://analytics.home-assistant.io/custom_integrations.json>.
Just ask your users to enable "Usage analytics" <https://www.home-assistant.io/integrations/analytics/#usage-analytics>

Reply:
Cool. I didn't know that HA Analytics also tracks custom integrations. I will remove the usage implementation.